infra/.env.example

```
@@ -20,22 +20,16 @@ POSTGRESQL_USER=postgres
-KC_HOSTNAME=keycloak.localhost
-KC_HTTP_PORT=8080
-KC_HTTPS_PORT=8443
-
 # ----------------------------
 # Backend Service
 # ----------------------------
 HOSTNAME=localhost
 BACKEND_URL=http://backend.localhost
-DATABASE_URL=postgresql://postgresql:5432/cvix
+# R2DBC URL for reactive database operations
+DATABASE_URL=r2dbc:postgresql://postgresql:5432/cvix
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the HOSTNAME key

(UnorderedKey)